src/framegrab/grabber.py

```diff
@@ -133,7 +148,7 @@ def grabbers_to_dict(grabber_list: list) -> dict:
         return grabbers
 
     @staticmethod
-    def create_grabber(config: dict, autogenerate_name: bool = True):
+    def create_grabber(config: dict, autogenerate_name: bool = True, warmup_delay: float = 1.0):
```

Comment:
I'd be fine leaving this out of the individual `create_grabber` method since it's handled in the batch create.